internal/pkg/config/env.go

```diff
@@ -29,18 +29,25 @@ type Environment struct {
 
 // CustomizeEnv represents the custom environment config.
 type CustomizeEnv struct {
-	ImportVPC *ImportVPC `json:"importVPC,omitempty"`
-	VPCConfig *AdjustVPC `json:"adjustVPC,omitempty"`
+	ImportVPC      *ImportVPC `json:"importVPC,omitempty"`
+	VPCConfig      *AdjustVPC `json:"adjustVPC,omitempty"`
+	ImportCertARNs []string   `json:"importCertARNs,omitempty"`
 }
 
-// NewCustomizeEnv returns a new CustomizeEnv struct.
-func NewCustomizeEnv(importVPC *ImportVPC, adjustVPC *AdjustVPC) *CustomizeEnv {
-	if importVPC == nil && adjustVPC == nil {
+// HasImportedCerts return if the environment has imported certs.
+func (e *Environment) HasImportedCerts() bool {
+	return e.CustomConfig != nil && len(e.CustomConfig.ImportCertARNs) != 0
+}
+
+// CustomizeEnvP returns a pointer to the CustomizeEnv value passed in.
+func CustomizeEnvP(input CustomizeEnv) *CustomizeEnv {
```

Comment:
out of curiousity, do we need the `CustomConfig` field's type to be `*CustomizeEnv` instead of `CustomizeEnv`? If we can make it a concrete type instead of a pointer then I think we can get rid of this func.

We can instead add 
```go
func (ce CustomizeEnv) IsEmpty() bool
```

Reply:
Unfortunately `omitempty` only works for nil struct (if the field has an empty value, defined as false, 0, a nil pointer, a nil interface value, and any empty array, slice, map, or string.) which means if we make it `CustomizeEnv` it will be shown in `env ls --json` as an empty struct if it is empty :(